ports/java/src/test/java/com/vmware/antlr4c3/TestCodeCompletionCore.java

```diff
@@ -20,16 +20,13 @@
 import org.antlr.v4.runtime.RecognitionException;
 import org.antlr.v4.runtime.Recognizer;
 import org.antlr.v4.runtime.atn.PredictionMode;
-import org.junit.FixMethodOrder;
 import org.junit.Test;
-import org.junit.runners.MethodSorters;
 
 import static org.junit.Assert.*;
 
 /**
  * Unit tests for CodeCompletionCore
  */
-@FixMethodOrder(MethodSorters.NAME_ASCENDING)
```

Comment:
This annotation was hiding the broken test. Test correctness shouldn't depend on the order in which the test methods are run.